### PR TITLE
LPS-102558,LPS-102561

### PR DESCRIPTION
**Pull request by julien**

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
ci:test:sf

**Comment by 4lejandrito** [comment]
Just started reviewing :)

File: modules/apps/document-library/document-library-preview-audio/src/main/resources/META-INF/resources/preview/view.jsp [hunk 1]
```diff
@@ -46,9 +26,40 @@ context.put("audioSources", audioSources);
...
 </liferay-util:html-top>
 
-<soy:component-renderer
-	componentId='<%= renderResponse.getNamespace() + randomNamespace + "previewAudio" %>'
-	context="<%= context %>"
-	module="preview/js/AudioPreviewer.es"
-	templateNamespace="com.liferay.document.library.preview.AudioPreviewer.render"
-/>
\ No newline at end of file
+<div>
```
Comment (boton) [inline modules/apps/document-library/document-library-preview-audio/src/main/resources/META-INF/resources/preview/view.jsp]: it's necessary this extra div 🤔  here and in video

**Comment by 4lejandrito** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/79277. [See changes here.](https://github.com/4lejandrito/liferay-portal/compare/7811d55b1fb6~5...7811d55b1fb6)

**Comment by julien** [comment]
thanks @4lejandrito

**Pull request closed without merging**